build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactmetrics/ArtifactSizeMetricsPlugin.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.sdk.kotlin.gradle.plugins.artifactmetrics
+
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.TaskProvider
+import org.gradle.jvm.tasks.Jar
+import org.gradle.kotlin.dsl.register
+import org.gradle.kotlin.dsl.withType
+
+private const val TASK_GROUP = "Metrics"
```

Comment:
**Nit**: Should be an existing task group like `"Verification"`

Reply:
Where are those task groups defined? I had made this comment earlier which caused Omar to change it to "Metrics"
https://github.com/awslabs/aws-kotlin-repo-tools/pull/34#discussion_r1546813068

Reply:
They're defined all over the place by various plugins and by Gradle itself. The simplest way to find the list of _existing_ groups is to run `./gradlew tasks`. For **aws-sdk-kotlin**, the current groups are:
* `application`
* `build`
* `build setup`
* `codegen`
* `deprecated`
* `distribution`
* `documentation`
* `formatting`
* `help`
* `ide`
* `publishing`
* `run`
* `verification`